Add an easy way to run a pool of workers (#3495)
os-autoinst/openQA: docker/README.md

Check whether the worker connected in the WebUI's administration interface.

> To launch a pool of workers could be used ./run.sh. This script will launch the

Reviewer: ```suggestion
To launch a pool of workers one could use the script `./run.sh`. It will launch the
```

> desired number of workers in individual containers and with the flag instance
> with consecutive numbers

Reviewer: ```suggestion
desired number of workers in individual containers using consecutive numbers for
the `--instance` parameter.
```

Author: Thanks
